### DISCUSSION TITLE
ppocrlabel标注 新的需求：如何将文本显示在坐标框起始点位置，这样方便一目了然，（ppocrlabel已经实现）

### DISCUSSION THREAD

**Discussion by gengyanlei** [PaddlePaddle/PaddleOCR#12758] [Q&A] [answered]
新的需求：如何将文本显示在坐标框起始点位置，这样方便一目了然，对比起来更方便。

**Comment by UserWangZz** [comment]
这个考虑到某些文本密集场景，暂不支持将文本显示在标注框附近

**Comment by GreatV** [comment]
ppocrlabel 已经移动到 https://github.com/PFCCLab/PPOCRLabel 可以到那里提需求

**Comment by gengyanlei** [comment]
好的。

**Comment by gengyanlei** [comment, marked as answer]
@GreatV @tink2123 
你们对自己写的代码都不了解么，我意外发现的，这个就是显示文本在标注框附件的功能（已经实现了）
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/22360785/30309ce1-d838-434c-847d-69dd86713193)

**Reply by SWHL** [reply]
我猜PPOCRLabel代码不仅仅是这两个同学写的，估计都换好几拨了，不知道也挺正常的。
